client/web/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+import com.github.gradle.node.npm.task.NpmTask
+
+plugins {
+    base
+    id("com.github.node-gradle.node")
+    distribution
+}
+
+node {
+    // Version of node to use.
+    version.set("16.13.0")
+
+    // Version of npm to use.
+    npmVersion.set("8.1.0")
+
+    // If true, it will download node using above parameters.
+    // If false, it will try to use globally installed node.
+    download.set(false)
+}
```

Comment:
This seems duplicated from `client/build.gradle.kts`

Reply:
Ok, If this doesn't cause an issue, I think we are Ok with this. @skawian do you have a view on this?